technical-reference/reference-technical-articles/xvs-bridge.md

```diff
@@ -212,3 +216,23 @@ In addition to the core functionality, the XVS Cross-chain Bridge includes addit
 - The transfer amount exceeds the single or daily receiving transaction limit.
 - The recipient is blacklisted by the XVS token.
 - The minting cap on the destination bridge is exceeded.
+
+### Retry Mechanism for Failed Transactions
+
+In the event of a failed transaction, follow a step-by-step process using block explorers and the `retryMessage` function to retry transactions on the respective blockchain. Here's a detailed guide:
+
+1. **Identify the Failed Transaction:**
```

Comment:
I would suggest using LayerZeroScan to identify the transaction in the target network, providing the transaction hash in the source network. The `MessageFailed` event is in the transaction sent to the target network